test/gtest/ucm/malloc_hook.cc

```diff
@@ -1046,12 +1046,13 @@ UCS_TEST_F(malloc_hook_cplusplus, remap_override_multi_threads) {
 typedef int (munmap_f_t)(void *addr, size_t len);
 
 UCS_TEST_SKIP_COND_F(malloc_hook, bistro_patch, RUNNING_ON_VALGRIND) {
+    const size_t max_patch_size = 32;
```

Comment:
needed to make test pass.
before this PR, the first 64bit of the patch were affected by hook address
after this PR, 64bit of the patch are same for all hooks (ldr x15 ; br x15) - need to compare more bytes so `EXPECT_NE(patched, origin);` in line 1088 will not fail